app/src/test/java/com/keylesspalace/tusky/ComposeActivityTest.kt

```diff
@@ -109,28 +111,14 @@ class ComposeActivityTest {
 
             override fun enqueue(callback: Callback<List<Emoji>>?) {}
         })
-        `when`(apiMock.instance).thenReturn(object: Call<Instance> {
-            override fun isExecuted(): Boolean {
-                return false
-            }
-            override fun clone(): Call<Instance> {
-                throw Error("not implemented")
-            }
-            override fun isCanceled(): Boolean {
-                throw Error("not implemented")
-            }
-            override fun cancel() {
-                throw Error("not implemented")
-            }
-            override fun execute(): Response<Instance> {
-                throw Error("not implemented")
-            }
-            override fun request(): Request {
-                throw Error("not implemented")
-            }
-
-            override fun enqueue(callback: Callback<Instance>?) {
-                instanceResponseCallback?.invoke(this, callback)
+        `when`(apiMock.instance).thenReturn(object: Single<Instance>() {
```

Comment:
Couldn't you just do `Single.just()` (or however it's called) instead of subclassing `Single`?

Reply:
Maybe? I'll take a look